diff --git a/dist/changesets-cli.cjs.js b/dist/changesets-cli.cjs.js
index 9f0bd6c5f96011be225269a71365a878cb822bb7..36210d53680656b0814de8a0258f431aca9958ff 100644
--- a/dist/changesets-cli.cjs.js
+++ b/dist/changesets-cli.cjs.js
@@ -741,28 +741,9 @@ function getCorrectRegistry(packageJson) {
   };
 }
 async function getPublishTool(cwd) {
-  const pm = await packageManagerDetector.detect({
-    cwd
-  });
-  if (!pm || pm.name !== "pnpm") return {
+  return {
     name: "npm"
   };
-  try {
-    let result = await spawn__default["default"]("pnpm", ["--version"], {
-      cwd
-    });
-    let version = result.stdout.toString().trim();
-    let parsed = semverParse__default["default"](version);
-    return {
-      name: "pnpm",
-      shouldAddNoGitChecks: (parsed === null || parsed === void 0 ? void 0 : parsed.major) === undefined ? false : parsed.major >= 5
-    };
-  } catch (e) {
-    return {
-      name: "pnpm",
-      shouldAddNoGitChecks: false
-    };
-  }
 }
 async function getTokenIsRequired() {
   const {
diff --git a/dist/changesets-cli.esm.js b/dist/changesets-cli.esm.js
index d9582bf53eaf55402a6bf96a39589af7854b8f23..2bd39fbe69c712a3ca43b73174970c067bf750a6 100644
--- a/dist/changesets-cli.esm.js
+++ b/dist/changesets-cli.esm.js
@@ -704,28 +704,9 @@ function getCorrectRegistry(packageJson) {
   };
 }
 async function getPublishTool(cwd) {
-  const pm = await detect({
-    cwd
-  });
-  if (!pm || pm.name !== "pnpm") return {
+  return {
     name: "npm"
   };
-  try {
-    let result = await spawn$1("pnpm", ["--version"], {
-      cwd
-    });
-    let version = result.stdout.toString().trim();
-    let parsed = semverParse(version);
-    return {
-      name: "pnpm",
-      shouldAddNoGitChecks: (parsed === null || parsed === void 0 ? void 0 : parsed.major) === undefined ? false : parsed.major >= 5
-    };
-  } catch (e) {
-    return {
-      name: "pnpm",
-      shouldAddNoGitChecks: false
-    };
-  }
 }
 async function getTokenIsRequired() {
   const {
